crates/optimism/chainspec/src/lib.rs

```diff
@@ -52,14 +55,14 @@ impl EthChainSpec for OpChainSpec {
         self.inner.chain()
     }
 
-    fn base_fee_params_at_timestamp(&self, timestamp: u64) -> BaseFeeParams {
-        self.inner.base_fee_params_at_timestamp(timestamp)
-    }
-
     fn base_fee_params_at_block(&self, block_number: u64) -> BaseFeeParams {
         self.inner.base_fee_params_at_block(block_number)
     }
 
+    fn base_fee_params_at_timestamp(&self, timestamp: u64) -> BaseFeeParams {
+        self.inner.base_fee_params_at_timestamp(timestamp)
+    }
+
```

Comment:
suggested lint was to reorg the methods in the same order as the trait definition